h2o-r/tests/testdir_munging/runit_pubdev_4870_sort_bug.R

```diff
@@ -0,0 +1,33 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+library(testthat)
+
+# test float sort with more than one columns.  A bug was found in PUBDEV-4870.
+test.sort = function() {
+
+  X = h2o.uploadFile(locate("smalldata/synthetic/smallIntFloats.csv.zip"))
+  sorted_column_names = c("C1",  "C10")
+  sorted_column_indices = c(1,2)
+  browser()
+  X_sorted = h2o.arrange(X, C1, C10)
+  check_sorted_two_columns(X_sorted, sorted_column_indices, prob=0.2)
+}
+
+check_sorted_two_columns <- function(frame, column_indices, prob=0.5) {
```

Comment:
Please see the comments in the Python counterpart. Same apply here

Reply:
Fixed